index.js

```diff
@@ -174,7 +174,7 @@
             instance,
             clickOutsideHandler,
             this.props.outsideClickIgnoreClass || IGNORE_CLASS,
-            this.props.excludeScrollbar || false,
+            this.props.excludeScrollbar,
```

Comment:
can you also add the comment that explains why this has no fallback? good for future code-readers to have